src/registers/sctlr_el3.rs

```diff
@@ -0,0 +1,585 @@
+        ///
+        /// 0b10    Tag Check Faults are asynchronously accumulated.
+        ///
+        /// 0b11    When FEAT_MTE3 is implemented:
+        ///           Tag Check Faults cause a synchronous exception on reads, and are asynchronously
+        ///           accumulated on writes.
+        ///
+        /// The reset behavior of this field is:
+        ///         • On a Warm reset, in a system where the PE resets into EL3, this field resets to an
+        ///         architecturally UNKNOWN value.
+        TCF OFFSET(40) NUMBITS(2) [
+            NoEffect = 0,
+            SyncException = 1,
+            AsyncAccumulated = 2,
+            SyncReadAsyncWrite = 3
+        ],
+
+        /// When FEAT_MTE2 is implemented:
+        ///
+        /// When synchronous exceptions are not being generated by Tag Check Faults, this field controls
+        /// whether on exception entry into EL3, all Tag Check Faults due to instructions executed before
+        /// exception entry, that are reported asynchronously, are synchronized into TFSRE0_EL1 and
+        /// TFSR_ELx registers.
+        ///
+        /// 0b0    Tag Check Faults are not synchronized on entry to EL3.
+        ///
+        /// 0b1    Tag Check Faults are synchronized on entry to EL3.
+        ///
+        /// The reset behavior of this field is:
+        ///        • On a Warm reset, in a system where the PE resets into EL3, this field resets to an
+        ///        architecturally UNKNOWN value.
+        ITFSB OFFSET(37) NUMBITS(1) [
+            NoSyncEntry = 0,
+            SyncEntry = 1
+        ],
+
+        /// When FEAT_BTI is implemented:
+        ///
+        /// PAC Branch Type compatibility at EL3.
+        ///
+        /// 0b0    When the PE is executing at EL3, PACIASP and PACIBSP are compatible with
+        ///        PSTATE.BTYPE == 0b11.
+        ///
+        /// 0b1    When the PE is executing at EL3, PACIASP and PACIBSP are not compatible with
+        ///        PSTATE.BTYPE == 0b11.
+        ///
+        /// The reset behavior of this field is:
+        ///        • On a Warm reset, in a system where the PE resets into EL3, this field resets to an
+        ///        architecturally UNKNOWN value.
+        BT OFFSET(36) NUMBITS(1) [
+            Compat = 0,
+            NoCompat = 1
+        ],
+
+        /// When FEAT_PAuth is implemented:
+        ///
+        /// Controls enabling of pointer authentication (using the APIAKey_EL1 key) of instruction addresses
+        /// in the EL3 translation regime.
+        ///
+        /// Possible values of this bit are:
+        ///
+        /// 0b0    Pointer authentication (using the APIAKey_EL1 key) of instruction addresses is not
+        ///        enabled.
+        ///
+        /// 0b1    Pointer authentication (using the APIAKey_EL1 key) of instruction addresses is
+        ///        enabled.
+        ///
+        /// For more information, see Pointer authentication on page D8-5164.
```

Comment:
What version of the Arm reference are you using? In ARM DDI 0487H.a this page is D5-4775

Reply:
Not sure, it might have been an older version.